.github/workflows/ci.yml

```diff
@@ -59,3 +59,14 @@ jobs:
         uses: coverallsapp/github-action@v2
         with:
           file: _build/test/covertool/dns_erlang.covertool.xml
+          parallel: true
+
+  finish:
+    needs: test
+    if: ${{ always() }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Close parallel build
+        uses: coverallsapp/github-action@v2
+        with:
+          parallel-finished: true
```

Comment:
This is great. We should add it to the rest of our repos.

Reply:
`erldns` already has it, added it a long time ago, dunno why it wasn't here. Will check the rest later 👌🏽